package-generator/Dockerfile

```diff
@@ -5,9 +5,9 @@ RUN apt-get update && sudo apt-get -y install python-pip python-dev \
   libffi-dev zip python-mysqldb mysql-server
 RUN git clone https://github.com/telefonicaid/fiware-enablers /opt/fiware-enablers
 WORKDIR /opt/fiware-enablers/package-generator
+RUN pip install nose pytz
```

Comment:
Ey Warning, it was the problem that was commented today by Jesús and myself... due to a problem with the version of pip, correct me if I am wrong @jesuspg